modules/social_features/social_user/social_user.install

```diff
@@ -152,3 +152,17 @@ function social_user_update_13000(): ?string {
   // Output logged messages to a related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Change search operator in views.view.user_admin_people.
+ */
+function social_user_update_12003(): string {
```

Comment:
@VBaldych Since we want to release it in version 12.2.x, better to rename hook update to `_12201`
Also, you can put it before `social_user_update_13000`

Reply:
@ribel done!